libs/html-tag/src/lib/html-tag/html-tag.component.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  KeyValueDiffers,
+  OnChanges,
+  OnDestroy,
+  Renderer2,
+  SimpleChanges,
+  TemplateRef,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import {
+  DynamicComponent,
+  OrchestratorConfigItem,
+  OrchestratorDynamicComponent,
+} from '@orchestrator/core';
+import { HtmlTagConfig } from './html-tag-config';
+
+@Component({
+  selector: 'orc-html-tag',
+  templateUrl: './html-tag.component.html',
+  styleUrls: ['./html-tag.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+@DynamicComponent({ config: HtmlTagConfig })
+export class HtmlTagComponent
+  implements OrchestratorDynamicComponent<HtmlTagConfig>, OnChanges, OnDestroy
+{
+  @Input() items?: OrchestratorConfigItem<unknown>[];
+  @Input() config?: HtmlTagConfig;
+
+  /** @internal */
+  @ViewChild('tagContentAnchor', { static: true, read: ViewContainerRef })
+  tagContentVcr?: ViewContainerRef;
```

Comment:
Minor: why not to start internal properties with `_` or something like this ?

Reply:
I think we can use `_` it's just not been used before like that.

Reply:
I've adjusted these internal prop names.